contracts/security/src/events/remove_security.rs

```diff
@@ -0,0 +1,70 @@
+use cosmwasm_std::{Addr, Attribute, Event};
+
+use crate::core::constants::{REMOVE_SECURITY_ASSET, REMOVE_SECURITY_EVENT};
+
+/// An event to represent removing the security from an asset.
+pub struct RemoveSecurityEvent {
+    pub asset_addr: Addr,
+}
+
+impl RemoveSecurityEvent {
+    /// Creates a new instance of RemoveSecurityEvent
+    ///
+    /// # Arguments
+    ///
+    /// * `asset_addr` - The address of the asset being modified.
+    ///
+    /// # Examples
+    /// ```
+    /// let asset_addr = Addr::unchecked("address");
+    /// let event = SetSecurityEvent::new(asset_addr);
+    /// ```
+    pub fn new(asset_addr: &Addr) -> Self {
+        Self {
+            asset_addr: asset_addr.clone(),
+        }
+    }
+}
+
+/// Allows us to easily convert a SetTagEvent into an Event.
+impl From<RemoveSecurityEvent> for Event {
+    fn from(val: RemoveSecurityEvent) -> Self {
+        let event = Event::new(REMOVE_SECURITY_EVENT);
+        event.add_attributes([Attribute::new(
+            REMOVE_SECURITY_ASSET,
+            val.asset_addr.to_string(),
```

Comment:
This only emits the asset addr but nothing about what security is being removed?